gradle.properties

```diff
@@ -18,5 +18,5 @@ ideaEdition = IC
 ideaVersion = 2017.1
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.2.2_2017-SNAPSHOT
+version = 17.2.3
```

Comment:
looks like the tag is good, so you can just push a commit to this PR without bothering with another commit